tuftool/Cargo.toml

```diff
@@ -12,14 +12,14 @@ edition = "2018"
 integ = []
 default = ["aws-sdk-rust"]
 aws-sdk-rust = ["aws-sdk-rust-rustls"]
-aws-sdk-rust-rustls = ["aws-config/rustls", "aws-sdk-ssm/rustls", "aws-sdk-kms/rustls", ]
+aws-sdk-rust-rustls = []
```

Comment:
This isn't quite right, you are turning off `default-features`, which disables the `rustls` support turned on by default in the `aws-sdk-ssm`. You should add back `aws-smithy-runtime/tls-rustls` to this list so that by default, `tls` isn't enabled, but users can still enable it by turning on ` aws-sdk-rust-rustls`. 

The same applies for `aws-sdk-kms`.

Reply:
The `default-https-client` feature enables the updated HTTPS stack with `hyper 1.x`, `rustls`, and `aws-lc-rs`, so disabling this feature doesn't actually disable support for `rustls`. This feature currently just enables the legacy HTTPS stack with `hyper 0.14.x` and `ring`. For more info see this announcement from `aws-sdk-rust`: https://github.com/awslabs/aws-sdk-rust/discussions/1257

We want to disable this feature to avoid needing to build two versions of `hyper` and `rustls` for the updated `aws-sdk` crates.